src/scss/components/ubication.scss

```diff
@@ -0,0 +1,58 @@
+.map-section {
+  padding-bottom: 19.2px;
+
+  &__pictureBotton {
+    background-image: url('../static/assets/Forma2-map-mobile.png');
+    background-position-x: 100%;
+    background-position-y: 100%;
+    background-repeat: no-repeat;
+    background-size: 98.1px 100.8px;
+  }
+
+  &__pictureTop {
```

Comment:
Remember, in BEM notation for compound words is used snake case, could you replace `pictureTop` to `picture-top`